### PR TITLE
Drop Pulp 2 requires from katello

### PR DESCRIPTION
**Pull request by ehelms**

### CODE REVIEW

**Review by ekohl — COMMENTED** [review]
We also have https://github.com/theforeman/foreman-packaging/blob/e0df5ab38228d8f10b5c1ed8342b9e72de294cf6/packages/katello/katello/katello.cron and I wonder if those are still needed in a Pulp 3 world.

Other than that, it looks good.

File: packages/katello/katello/katello.spec [hunk 1]
```diff
@@ -5,7 +5,7 @@
 %global confdir common
 %global prereleasesource master
 %global prerelease %{?prereleasesource:.}%{?prereleasesource}
-%global release 7
+%global release 8
 
 Name:       katello
 Version:    4.0.0
```
Comment (ekohl) [inline packages/katello/katello/katello.spec:11]: Aren't we at 4.1.0 by now?

Comment (ehelms) [reply packages/katello/katello/katello.spec:11]: I think folks are working on that, but this PR is not the one to change the version. And this change applies to 4.0 as well.

**Comment by evgeni** [comment]
seems like it's something that exists in pulp3 too: https://github.com/katello/katello/blob/master/app/lib/actions/katello/orphan_cleanup/remove_orphans.rb

(not sure that code will not explode w/o pulp2, but that's a different question)

**Review by evgeni — APPROVED** [review]

**Pull request merged**